libpod/container.go

```diff
@@ -383,6 +383,11 @@ func (c *Container) setupStorage() error {
 	c.config.StaticDir = containerInfo.Dir
 	c.state.RunDir = containerInfo.RunDir
 
+	artifacts := filepath.Join(c.config.StaticDir, "artifacts")
```

Comment:
as we've the "artifacts" string in two places, I'd like to be ultra safe and have a constant declared for it.

Reply:
fixed